src/compiler/scanner.ts

```diff
@@ -270,9 +270,14 @@ namespace ts {
 
     /**
-     * Test for whether a comment's text contains a `@ts-expect-error` or `@ts-ignore` directive.
+     * Test for whether a single line comment's text contains a directive.
      */
-    const commentDirectiveRegEx = /^\s*\/\/\/?\s*@(ts-expect-error|ts-ignore)/;
+    const commentDirectiveRegExSingleLine = /^\s*\/\/\/?\s*@(ts-expect-error|ts-ignore)/;
+
+    /**
+     * Test for whether a multi-line comment's last line contains a directive.
+     */
+    const commentDirectiveRegExMultiLine = /^\s*(?:\/\*)?\s*@(ts-expect-error|ts-ignore)/;

```

Comment:
So this doesn't work for

```ts
/**
 * @ts-expect-error */
```

?
```

Reply:
@DanielRosenwasser that's correct, by design right now. Would you like it to?

Reply:
Hmm... this also doesn't work for `/** @ts-ignore */` either, right?

Reply:
I also am a little surprised that this wouldn't work for

```ts
let y = <div>
  {/*@ts-ignore*/}
  <MyComponent foo={100} />;
</div>;
```

Reply:
But it might be out of scope for this PR

Reply:
Got it - should be resolved as of 199d256.